humanmade/hm-post-history
# 530: Bug - Update The Revisions Widget

## Issue Number
Issue humanmade/hm-playbook-theme#530

## Description
This change updates the Revisions Widget style bugs and date format.

| Default Style | Hover & Focus |  Selected Revision |
| ----------- | ----------- | ----------- |
| <img width="211" alt="Screen Shot 2022-12-23 at 2 28 04 PM" src="https://user-images.githubusercontent.com/14046668/209411326-4789287d-8e9f-4e52-bc88-69175b5ad188.png">     | <img width="210" alt="Screen Shot 2022-12-23 at 2 28 18 PM" src="https://user-images.githubusercontent.com/14046668/209411338-07df3add-b0f4-485e-8958-6a348b136117.png">      |<img width="209" alt="Screen Shot 2022-12-23 at 2 28 39 PM" src="https://user-images.githubusercontent.com/14046668/209411351-7ddfa16f-0f24-45a6-af0c-50f6b47359ba.png">     |

## Steps to Test
1. Navigate to a page with a Revision widget.
2. Test the "Load Revision" button and make sure it displays with a border and color background on focus and hover.
3. Check that the date is displayed in YYYY-MM-DD format.

## Acceptance Criteria

- [x] The widget uses YYYY-MM-DD to align with dates formats in the rest of the Playbook
- [x] ~The widget does not include the time of the revision~ Clean up the styling for this to make it more readable 
- [x] There is appropriate spacing between the last revision event and the `Load Revisions` CTA within the widget.
- [x] Has this been tested in multiple browsers
- [x] Has it been accessibility tested
- [x] Have all linters and PHPCS checks been run
- [x] Does the README.md and the Wiki need updating to reflect these changes
- [x] Do any articles on how to update the Playbook in https://playbook.hmn.md/category/updating-the-playbook/ need updating to reflect these changes.
